examples/rust-examples/xtensa-esp32/scheduler/src/main.rs

```diff
@@ -0,0 +1,50 @@
+#![no_std]
```

Comment:
Needs readme file for this example. And needs such example for esp32c6 architecture